src/pages/_app.tsx

```diff
@@ -558,8 +559,9 @@ export interface FlashMessage {
 export const AppContext = createContext<AppContextType>(null);
 
 const redirectMap = {
-    roadmap: (token: string) =>
+    roadmap: async (token: string) =>
```

Comment:
redirect to families is not working on `testing.ente.io`  
but redirect to roadmap is 
will have to check this once